packages/react-link/src/common/isConformant.ts

```diff
@@ -2,9 +2,9 @@ import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui
 
 export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'>) {
   const defaultOptions = {
-    disabledTests: [`has-docblock`, `kebab-aria-attributes`],
-
+    asPropHandlesRef: true,
     componentPath: module!.parent!.filename.replace('.test', ''),
+    disabledTests: [`has-docblock`, `kebab-aria-attributes`],
```

Comment:
Just wondering, is "kebab-aria-attributes" required there? 🐱

Reply:
You're right, this is not needed, I've removed it for now.